applications/multilingual-search.md

```diff
@@ -0,0 +1,18 @@
+---
+description: Machine translation for multilingual search
+---
+
+# Multilingual search
+
+Machine translation for **multilingual search** is the automatic translation of website content at the time of its indexing. The purpose of indexing is to help users retrieve the data they want. When search indexes are translated, users looking for information in different languages, or even alphabets, find relevant results.
```

Comment:
It could be either machine-translating the query or machine-translating the index.

So we better say what it's for - so users can search in one language and get results in another.

And then the 2 ways of implementing it can be mentioned after that.